src/wasm/wasm-binary.cpp

```diff
@@ -1117,9 +1119,10 @@ static void writeBase64VLQ(std::ostream& out, int32_t n) {
     }
     // more VLG digit will follow -- add continuation bit (0x20),
     // base64 codes 'g'..'z', '0'..'9', '+', '/'
-    out << char(digit < 20
-                  ? 'g' + digit
-                  : digit < 30 ? '0' + digit - 20 : digit == 30 ? '+' : '/');
+    out << char(digit < 20    ? 'g' + digit
+                : digit < 30  ? '0' + digit - 20
+                : digit == 30 ? '+'
+                              : '/');
```

Comment:
Did you do this automatically, or manually? `clang-format` errors on CI, so if this was automatic, maybe your version of `clang-format` is much newer or older..?

Reply:
I use clang-format automatically. it may caused by the version or some global setting. I will revert it.